RHDEVDOCS-4270: Added updates for Helm Releases and Repositories (#49737)
openshift/openshift-docs: modules/odc-installing-helm-charts-using-developer-perspective.adoc

+
.. Click *Install* to create a Helm release. You will be redirected to the *Topology* view where the release is displayed. If the Helm chart has release notes, the chart is pre-selected and the right panel displays the release notes for that release.

> .. View the newly created Helm release in the Helm Releases list page.

Reviewer: ```suggestion
.. View the newly created Helm release in the *Helm Releases* page.
```
* Bold because it is a UI label.

Author: Fixed.

> .. Verify that the Helm release is created by navigating to *+Add* in the Developer perspective and clicking *Actions*. In the *Helm Releases* page, you can view the details of the Helm release created in your project.

Reviewer 2: ```suggestion
.. Verify that in the *Helm Releases* list page, you can view the newly created Helm release.
```

Author: Hi @debsmita1 - The suggestion that you have provided in terms of a change in the language is looking incorrect grammatically. May I suggest another change, which can address what you are trying to say? Can it be as follows:

Check to see if you can view the newly created Helm release in the Helm Releases page.

Reviewer 2: `You should be able to view the newly created Helm release in the Helm Releases list page.` is good

Author: @debsmita1 - I have made the changes as per your suggestion. Kindly have a look at this change and let me know if it looks fine.

Author: Fixed. @debsmita1 - Sending a gentle reminder to you for your final validation of my changes.
